### PR TITLE
Adapt integration tests to reuse the packed charm

### PR DESCRIPTION
**Pull request by arturo-seijas**
Adapt the integration tests to reuse the exisintg charm artifact as per https://github.com/canonical/operator-workflows/pull/162 and fix the docker build:
* Install node using the nvm package manager to avoid installing the lsb_release package, which is failing to install due to issues with the docker storage driver
* Remove the python2-saml package, which shouldn't be needed, which is failing to install due to issues with the docker storage driver

### CODE REVIEW

**Review by jdkandersson — DISMISSED** [review]

**Review by weiiwang01 — DISMISSED** [review]

**Review by weiiwang01 — DISMISSED** [review]

**Review by yanksyoon — DISMISSED** [review]
LGTM!

**Review by weiiwang01 — DISMISSED** [review]

**Review by yanksyoon — DISMISSED** [review]

**Review by yanksyoon — DISMISSED** [review]

**Review by weiiwang01 — DISMISSED** [review]

**Review by weiiwang01 — DISMISSED** [review]

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = unit, integration
+envlist = unit, static, coverage-report
```
Comment (merkata) [inline tox.ini:7]: lint is out of scope I guess

**Review by merkata — DISMISSED** [review]
LGTM

File: .woke.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+rules:
+  # Ignore "whitelist" and "master".
+  - name: whitelist
+  - name: master
```
Comment (yanksyoon) [inline .woke.yml:4]: Out of curiosity, are these words included in this repository?

**Review by yanksyoon — DISMISSED** [review]
LGTM!

File: tox.ini (hunk 1 above)
Comment (arturo-seijas) [reply tox.ini]: I've added it just now. Otherwise the wf is not passing :(

**Review by merkata — DISMISSED** [review]

File: .woke.yml (hunk 2 above)
Comment (arturo-seijas) [reply .woke.yml:4]: Yes, they are

**Review by merkata — APPROVED** [review]

**Review by nrobinaubertin — APPROVED** [review]

**Review by weiiwang01 — APPROVED** [review]

**Pull request merged**